formalmethods_intrepid_app_upload_102.py

```
@@ -0,0 +1,36 @@
+        name = '__ctx{}'.format(len(contexts))
+        contexts[name] = {'context': ctx,
+                          'inputs': ctx.inputs,
+                          'nets': ctx.nets,
+                          'latches': ctx.latches,
+                          'engines': {},
+                          'traces': {},
+                          'simulators': {}}
+        return {'result': {'ctx': name}}, 201
+    except ParseError as err:
+        return {'result': err.message()}, 400
+    except:
```

Ruff (0.13.1)

Do not use bare `except`

(E722)